sdk/src/quic.rs

```diff
@@ -2,7 +2,7 @@ pub const QUIC_PORT_OFFSET: u16 = 6;
 // Empirically found max number of concurrent streams
 // that seems to maximize TPS on GCE (higher values don't seem to
 // give significant improvement or seem to impact stability)
-pub const QUIC_MAX_CONCURRENT_STREAMS: usize = 2048;
+pub const QUIC_MAX_UNSTAKED_CONCURRENT_STREAMS: usize = 128;
```

Comment:
What impact does this have on TPU clients?  Are there some stats about TPS or network performance with and without this change?

Reply:
I do not see a huge impact of w/o this change in my tpu-client test:

With this branch

2022-06-17T20:47:16.938332781Z INFO  solana_bench_tps::bench] ---------------------+---------------+--------------------
[2022-06-17T20:47:16.938343148Z INFO  solana_bench_tps::bench] http://35.247.120.58:8899 |      67938.42 | 1213244 
[2022-06-17T20:47:16.938348907Z INFO  solana_bench_tps::bench] 
    Average max TPS: 67938.42, 0 nodes had 0 TPS
[2022-06-17T20:47:16.938360601Z INFO  solana_bench_tps::bench] 
    Highest TPS: 67938.42 sampling period 1s max transactions: 1213244 clients: 1 drop rate: 0.60
[2022-06-17T20:47:16.938367384Z INFO  solana_bench_tps::bench] 	Average TPS: 19827.145


[2022-06-17T20:55:40.669179958Z INFO  solana_bench_tps::bench] http://35.247.120.58:8899 |      55039.97 | 1311307 
[2022-06-17T20:55:40.669184817Z INFO  solana_bench_tps::bench] 
    Average max TPS: 55039.97, 0 nodes had 0 TPS
[2022-06-17T20:55:40.669194603Z INFO  solana_bench_tps::bench] 
    Highest TPS: 55039.97 sampling period 1s max transactions: 1311307 clients: 1 drop rate: 0.29
[2022-06-17T20:55:40.669198742Z INFO  solana_bench_tps::bench] 	Average TPS: 21661.62


[2022-06-18T01:07:46.093266884Z INFO  solana_bench_tps::bench]  Node address        |       Max TPS | Total Transactions
[2022-06-18T01:07:46.093280556Z INFO  solana_bench_tps::bench] ---------------------+---------------+--------------------
[2022-06-18T01:07:46.093285691Z INFO  solana_bench_tps::bench] http://35.247.120.58:8899 |      58969.85 | 1367411 
[2022-06-18T01:07:46.093301930Z INFO  solana_bench_tps::bench] 
    Average max TPS: 58969.85, 0 nodes had 0 TPS
[2022-06-18T01:07:46.093292778Z INFO  solana_metrics::metrics] datapoint: bench-tps-lamport_balance balance=14373355930776120i
[2022-06-18T01:07:46.093311329Z INFO  solana_bench_tps::bench] 
    Highest TPS: 58969.85 sampling period 1s max transactions: 1367411 clients: 1 drop rate: 0.76
[2022-06-18T01:07:46.093349226Z INFO  solana_bench_tps::bench] 	Average TPS: 22578.842


Master


[2022-06-17T21:33:02.909510265Z INFO  solana_bench_tps::bench]  Node address        |       Max TPS | Total Transactions
[2022-06-17T21:33:02.909517556Z INFO  solana_bench_tps::bench] ---------------------+---------------+--------------------
[2022-06-17T21:33:02.909520144Z INFO  solana_bench_tps::bench] http://35.247.120.58:8899 |      44006.95 | 1102674 
[2022-06-17T21:33:02.909535908Z INFO  solana_bench_tps::bench] 
    Average max TPS: 44006.95, 0 nodes had 0 TPS
[2022-06-17T21:33:02.909548109Z INFO  solana_bench_tps::bench] 
    Highest TPS: 44006.95 sampling period 1s max transactions: 1102674 clients: 1 drop rate: 0.35
[2022-06-17T21:33:02.909526414Z INFO  solana_metrics::metrics] datapoint: bench-tps-lamport_balance balance=14373355930776120i
[2022-06-17T21:33:02.909555115Z INFO  solana_bench_tps::bench] 	Average TPS: 18090.404




[2022-06-17T21:38:51.706520412Z INFO  solana_bench_tps::bench] Token balance: 14373355930776120
[2022-06-17T21:38:51.706553141Z INFO  solana_bench_tps::bench]  Node address        |       Max TPS | Total Transactions
[2022-06-17T21:38:51.706559304Z INFO  solana_bench_tps::bench] ---------------------+---------------+--------------------
[2022-06-17T21:38:51.706562073Z INFO  solana_bench_tps::bench] http://35.247.120.58:8899 |      65728.62 | 1300790 
[2022-06-17T21:38:51.706573521Z INFO  solana_bench_tps::bench] 
    Average max TPS: 65728.62, 0 nodes had 0 TPS
[2022-06-17T21:38:51.706581605Z INFO  solana_bench_tps::bench] 
    Highest TPS: 65728.62 sampling period 1s max transactions: 1300790 clients: 1 drop rate: 0.34
[2022-06-17T21:38:51.706595604Z INFO  solana_bench_tps::bench] 	Average TPS: 21306.563